packages/core/src/domain/tracekit.ts

```diff
@@ -879,15 +879,15 @@ export const computeStackTrace = (function computeStackTraceWrapper() {
    * @param {(string|number)=} depth
    * @memberof computeStackTrace
    */
-  function doComputeStackTrace(ex: Error, depth?: string | number): StackTrace {
+  function doComputeStackTrace(ex: unknown, depth?: string | number): StackTrace {
```

Comment:
Because we are using `ex.name` and `ex.message` without checking if `ex` is an object, I would be a bit more restrictive on the typing:
```suggestion
  function doComputeStackTrace(ex: { name?: string, message?: string }, depth?: string | number): StackTrace {
```
This would avoid the cast to `any` below.

If we want `ex` to be really any value, then we should not access `ex.name` and `ex.message` directly.

Reply:
I've updated the strategy and kept the unknown.